Sources/JAYSON/JSON.swift

```diff
@@ -343,6 +354,94 @@ extension JSON {
       return false
     }
   }
+
+  /**
+   Returns a Boolean value that indicates if the key contains a value (including NSNull).
+
+   The following JSONs return true from finding key `name`.
+   ```json
+   {
+     "name": "John"
+   }
+   ```
+
+   ```json
+   {
+     "name": null
+   }
+   ```
+   */
+  public func contains(_ key: String...) -> Bool {
+    contains(key)
+  }
+
+  /**
+   Returns a Boolean value that indicates if the key contains a value (including NSNull).
+
+   The following JSONs return true from finding key `name`.
+   ```json
+   {
+     "name": "John"
+   }
+   ```
+
+   ```json
+   {
+     "name": null
+   }
+   ```
+   */
+  public func contains(_ keys: [String]) -> Bool {
+    var _source: Any = self.source
+    for key in keys {
+      guard let dictionary = (_source as? NSDictionary) else {
```

Comment:
I find it easier to read with a comma operator:
```
      guard
        let dictionary = (_source as? NSDictionary),
        let nextSource = dictionary.value(forKey: key)
      else {
        return false
      }
```

Reply:
Do you have a habit of putting parenthesis on casts ? I used to always do it in `C` but usually not in `Swift`

Reply:
thanks, I'll brush up this code.

> Do you have a habit of putting parenthesis on casts ? I used to always do it in C but usually not in Swift

No, I just didn't care about it. lol.
basically I think not using parenthesis is better in swift.